src/qt/rpcconsole.cpp

```diff
@@ -687,7 +687,12 @@ void RPCConsole::setClientModel(ClientModel *model)
         startExecutor();
     }
     if (!model) {
-        // Client model is being set to 0, this means shutdown() is about to be called.
+        // Client model is being set to nullptr, this means shutdown() is about to be called.
+        if (gArgs.GetBoolArg("-server", false)) {
+            InterruptRPC();
```

Comment:
This feels to be the wrong place to call shutdown/init functions. Shouldn't this be called through bitcoin.cpp?

Reply:
[Done](https://github.com/bitcoin/bitcoin/pull/18452#issuecomment-611024400).